routes/ordersRouter.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const { getAllOrders, getOrderById, getOrdersByUser } = require('../db/orders');
+const ordersRouter = express.Router();
+
+ordersRouter.get('/', async(req, res, next) => {
+	try{
+		const orders = await getAllOrders();
+		res.send(orders);
+	}
+	catch(error){
+		next(error);
+	}
+});
+
+ordersRouter.get('/:id', async(req, res, next) => {
+	const { id } = req.params;
+	try{
+		const order = await getOrderById(id);
```

Comment:
REQUIRED - Should be only sending back the order if `order.userId` matches the `req.user.id` (or if you're an admin :)

Reply:
edited, tested, functioning as requested.